### PR TITLE
Add warning about expiration of FCAP code

### PR DESCRIPTION
**Pull request by francois-berder-imgtec**
Signed-off-by: Francois Berder <Francois.Berder@imgtec.com>

### CODE REVIEW

File: src/cgi-bin/strings_en_GB.lua [hunk 1]
```diff
@@ -173,7 +173,7 @@ STR_DEVICE_REG_CODE = "Registration code"
 STR_REGISTER_CI40_TITLE = "Register your Ci40"
 STR_INVALID_DEVICE_REG_CODE_MSG = "Please enter a registration code.\n\n"
 STR_REGISTER = "Register"
-STR_REGISTER_CI40_MSG = "To manage your devices via FlowCloud register them with your account by retrieving a registration code from the web site and then entering it below. You can also identify your device by giving it a name and description."
+STR_REGISTER_CI40_MSG = "To manage your devices via FlowCloud register them with your account by retrieving a registration code from the web site and then entering it below. Provisioning with an FCAP code is only valid for 24 hours after generation. If the 24 hour window has closed please regenerate your code. You can also identify your device by giving it a name and description."
```
Comment (abhijit-mahajani-imgtec) [inline src/cgi-bin/strings_en_GB.lua]: Add comma.
If the 24 hour window has closed, please re-generate your code.

**Comment by abhijit-mahajani-imgtec** [comment]
Verified.

**Comment by abhijit-mahajani-imgtec** [comment]
Approved and Verified.

**Comment by nikhil-zinjurde-imgtec** [comment]
QA verified

**Pull request merged**